packages/cli/commands/cms/lighthouseScore.js

```diff
@@ -49,7 +49,10 @@ const selectTheme = async accountId => {
       message: i18n(`${i18nKey}.info.promptMessage`),
       choices: async () => {
         try {
-          const result = await fetchThemes(accountId);
+          const result = await fetchThemes(accountId, {
+            limit: 50,
```

Comment:
Default limit is 100 😆 we can just make this 500 for now, and then work the offset logic in later